appcues/src/main/java/com/appcues/debugger/AppcuesDebuggerManager.kt

```diff
@@ -44,14 +45,16 @@ internal class AppcuesDebuggerManager(context: Context, private val koinScope: S
         }
     }
 
-    fun start(activity: Activity, deepLinkPath: String? = null) = activity.runOnUiThread {
+    fun start(activity: Activity, mode: DebugMode) = activity.runOnUiThread {
+        val changingMode = this.mode != mode
         this.currentActivity = activity
+        this.mode = mode
         coroutineScope.coroutineContext.cancelChildren()
         // it is possible to re-enter start without a stop (deepLinks) - in which case we continue to
         // use the VM we already have - else, make new one here
         val viewModel = debuggerViewModel ?: DebuggerViewModel(koinScope)
         debuggerViewModel = viewModel // and save reference
-        viewModel.onStart(deepLinkPath)
+        viewModel.onStart(mode, changingMode)
```

Comment:
Maybe a better way to handle, but at some level we need to know if we are `changingMode` (i.e. going from `Debugger` to `ScreenCapture` or vice-versa).  This is so we can force reset the mode down in the composition where it controls the FAB display.